src/create-rpc-client/index.ts

```
@@ -22,37 +22,40 @@ switch (readSystemId()) {
 }
 
 export async function createRpcClient(): Promise<RpcApi> {
-  let idToken: string;
-
+  let idToken: string | undefined;
   try {
     idToken = await getAccessToken();
-    await checkLoggedIn();
   } catch (e) {
-    throw new TerseError('Please login to continue');
+    idToken = undefined;
```

I'm revisiting this PR and am noticing that we are discarding the exception `e` here without doing any kind of check on what the error actually is. For example a coding mistake somewhere in `getAccessToken` could also throw an exception. If there is a specific scenario in which we expect getAccessToken to throw, let's check for it. It can cause very difficult-to-diagnose errors if we catch the exception and then don't do anything with it.